dnd5esheets/front/src/store/index.ts

```diff
@@ -199,6 +198,17 @@ const effects = {
     ])
   ),
 
+  // Recompute the passive perception score when the character's wisdom changes
+  ...Object.fromEntries(
+    [
+      "wisdom"
+    ].map((attribute) => [
+      `passive_perception`,
+      (character: CharacterSchema) =>
+        10 + scoreToProficiencyModifier(character.data.wisdom, character.data.proficiencies.perception, character.data.proficiency_bonus)
+    ])
+  ),
```

Comment:
You don't need `Object.fromEntries` here. I used it in some other places to simplify a bit, but here you can go straight with:
```suggestion
    passive_perception: (character: CharacterSchema) =>
        10 + scoreToProficiencyModifier(character.data.wisdom, character.data.proficiencies.perception, character.data.proficiency_bonus)
```

Reply:
Nice, indeed, it reads better!